src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function calendarWidth($cell-size, $gap, $padding) {
+  @return $cell-size * 7 + $gap * 6 + $padding * 2;
```

Comment:
Accept 7 as an argument, 6 can be calculated from it.

Reply:
don't get it 🥲

Reply:
Ok, what is 7? It's `$cell-count`. So you can add it to arguments (just pass it to this function where you call it).
6 = `$cell-count - 1`

Reply:
Rework this function